locales/de/common.json

```diff
@@ -57,77 +57,77 @@
-  "Nothing changed": "Keine Änderung",
+  "Nothing changed": "Keine Änderungen vorgenommen",
   "No change was necessary": "Es waren keine Änderungen nötig",
-  "Failed to fix NXM handling in Chrome": "Konnte Behandlung von NXM links in Chrome nicht reparieren",
+  "Failed to fix NXM handling in Chrome": "Konnte Verwendung von NXM-Links in Chrome nicht reparieren",
   "Report": "Melden",
   "Success": "Erfolg",
   "Fix was applied": "Erfolgreich repariert",
   "Categories": "Kategorien",
   "Plugins": "Plugins",
   "Save Games": "Spielstände",
   "Show downloads from other games": "Downloads für andere Spiele anzeigen",
-  "Savegame id": "Spielstand id",
+  "Savegame id": "Spielstand-ID",
   "Creation Time": "Zeitpunkt d. Speicherns",
   "Delete": "Löschen",
   "Browse": "Suchen",
   "Meta server": "Metaserver",
-  "Servers to query for meta data.": "Server auf denen nach Metadaten gesucht wird.",
+  "Servers to query for meta data.": "Server, auf denen nach Metadaten gesucht wird.",
   "Add a metaserver": "Metaserver hinzufügen",
   "{{count}} selected": "{{count}} ausgewählt",
-  "Do you really want to delete this mod?": "Wollen Sie diese Mod löschen?",
-  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods löschen?",
+  "Do you really want to delete this mod?": "Wollen Sie diese Mod wirklich löschen?",
+  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods wirklich löschen?",
   "Remove Mod": "Mod löschen",
   "Remove Archive": "Archiv löschen",
   "Disable Dependent": "Abhängige Mods deaktivieren",
-  "Confirm deletion": "Löschung bestätigen",
-  "Confirm": "Fortfahren",
+  "Confirm deletion": "Löschen bestätigen",
+  "Confirm": "Bestätigen",
   "Third-party libraries": "Bibliotheken von Drittanbietern",
-  "License": "Lizens",
+  "License": "Lizenz",
   "Released under": "Veröffentlicht unter",
   "External Changes": "Externe Änderungen",
-  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Dateien wurden geändert seit Vortex zuletzt aktiviert hat. Sie müssen entscheiden was damit passiert.",
+  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Eine oder mehrere Dateien wurden seit dem letzten Start von Vortex verändert. Sie müssen bestimmen, was mit ihnen passiert.",
   "These files were modified": "Diese Dateien wurden verändert",
-  "These files were replaced (or removed from the mod)": "Diese Dateien wurden ersetzt (oder in der Mod gelöscht)",
+  "These files were replaced (or removed from the mod)": "Diese Dateien wurden ersetzt (oder aus der Mod entfernt)",
   "These files were deleted": "Diese Dateien wurden gelöscht",
   "Cancel activation": "Aktivierung abbrechen",
   "Confirm changes": "Änderungen bestätigen",
   "Restore": "Wiederherstellen",
   "Category": "Kategorie",
-  "Endorsed": "Endorsed",
+  "Endorsed": "Zu empfehlen",
```

Comment:
Würde ich lieber un-übersetzt lassen.
Erstmal ist "endorsed" eben der Begriff der auf Nexusmods verwendet wird und dort gibt es keine Übersetzung.
Wenn wir das in Vortex übersetzen könnte das Verwirrung stiften. Den englischen Begriff beibehalten ist zwar unschön aber weniger missverständlich.
Außerdem klingt "Zu empfehlen" als würden wir dem Nutzer diese Mod empfehlen, tatsächlich geht es aber um seine Empfehlung an andere.